src/sentry/event_manager.py

```diff
@@ -384,6 +379,9 @@ def save(self, project, raw=False):
         release = data.pop('release', None)
         environment = data.pop('environment', None)
 
+        # unused
+        time_spent = data.pop('time_spent', None)
```

Comment:
Does this need to be preserved in a variable? Or can we just pop without leaving the used var floating around?